### PR TITLE
Add SOCA logo and info

### PR DESCRIPTION
**Pull request by Socaverse**
Add SOCA logo and info #17915

### CODE REVIEW

**Comment by Cryptocool1** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 10,000 holder addresses and 15,000 transactions, in order to filter out scam tokens.
See the criteria at : https://developer.trustwallet.com/assets/requirements .

**Comment by Socaverse** [comment]
noooo pls, what you told me was 5000 holders last month

**Comment by Socaverse** [comment]
@Cryptocool1

**Comment by Cryptocool1** [comment]
hello!
did you made a payment in you previous PR?

**Comment by Socaverse** [comment]
yes

**Comment by Socaverse** [comment]
https://github.com/trustwallet/assets/pull/17915

**Comment by Socaverse** [comment]
it shows 'All checks have failed'
what's the problem

**Comment by Cryptocool1** [comment]
Token check error:
❌ ID is not a valid Ethereum address
❌ explorer field incorrect: should be standard https://bscscan.com/token/https://bscscan.com/token/0x57af121a8ddb1f9a8fafcf3229c92bf2856a8a29, not https://bscscan.com/token/0x78c6E90aFa87E2C97cac1A07652Aa0998C6C9078
❌ invalid value 'https://www.coingecko.com/en/coins/socaverse' for coingecko link url, allowed only with prefix: https://coingecko.com/

**Comment by Socaverse** [comment]
it's a bsc address

File: blockchains/smartchain/assets/0x57Af121A8dDb1F9A8FafcF3229c92bF2856A8a29/info.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Socaverse",
+    "type": "BEP20",
+    "symbol": "SOCA",
+    "decimals": 9,
+    "website": "https://www.socaverse.one/",
+    "description": "A soccer community with NFTs, Farms, DeFi, and sports betting platfrom.",
+    "explorer": "https://bscscan.com/token/0x78c6E90aFa87E2C97cac1A07652Aa0998C6C9078",
+    "status": "active",
+    "id": "https://bscscan.com/token/0x57af121a8ddb1f9a8fafcf3229c92bf2856a8a29",
```
Comment (Cryptocool1) [inline blockchains/smartchain/assets/0x57Af121A8dDb1F9A8FafcF3229c92bF2856A8a29/info.json:10]: id should be 0x78c6E90aFa87E2C97cac1A07652Aa0998C6C9078

**Comment by Socaverse** [comment]
i fiexed them but i dont know how to replace the file

**Comment by Socaverse** [comment]
0x57af121a8ddb1f9a8fafcf3229c92bf2856a8a29
this is the token address

**Comment by Cryptocool1** [comment]
why are you add this explorer? https://bscscan.com/token/0x78c6E90aFa87E2C97cac1A07652Aa0998C6C9078

**Comment by Socaverse** [comment]
sorry it's a mistake

**Comment by Socaverse** [comment]
https://bscscan.com/token/0x57af121a8ddb1f9a8fafcf3229c92bf2856a8a29
this one is correct

**Comment by Socaverse** [comment]
the id should be 0x57Af121A8dDb1F9A8FafcF3229c92bF2856A8a29
but there's a excess'9' in the end, and i tried so many times can't fix it

**Comment by Cryptocool1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

**Comment by Socaverse** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
A community for soccer lovers
• How is the token being distributed? Are you conducting an airdrop?
https://soca.gitbook.io/socaverse/usdsoca/token-distribution
• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
Presale link:https://www.pinksale.finance/#/launchpad/0x6BFaF7537290e9cB0848933Bc63425C113B1F61a?chain=BSC
• Is the token listed on a centralized/decentralized exchange?
Not yet
• Is a Smart Contract audit completed? Where can we find this information?
Audit:https://www.rugfreecoins.com/details/7306

**Comment by Socaverse** [comment]
> 

• What is the purpose of this token? How will it benefit Trust Wallet users?
A community for soccer lovers
• How is the token being distributed? Are you conducting an airdrop?
https://soca.gitbook.io/socaverse/usdsoca/token-distribution
• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
Presale link:https://www.pinksale.finance/#/launchpad/0x6BFaF7537290e9cB0848933Bc63425C113B1F61a?chain=BSC
• Is the token listed on a centralized/decentralized exchange?
Not yet
• Is a Smart Contract audit completed? Where can we find this information?
Audit:https://www.rugfreecoins.com/details/7306

**Comment by Socaverse** [comment]
@Cryptocool1

**Pull request merged**